src/DebugReport.cpp

```diff
@@ -230,15 +230,24 @@ void DebugReporter::generateDebugReport(
     DebugReportSection topsortSCCGraphSection =
             getCodeSection(id + "-topsort-scc-graph", "SCC Topological Sort Order", topsortSCCGraph.str());
 
+    // TODO (azreika): adornment output temporarily disabled; should only be computed just before MST
```

Comment:
Could you remove this code instead of commenting it out? When it's time to re-enable, it can be merged back in.